modules/mono/editor/GodotTools/GodotTools/Build/BuildManager.cs

```diff
@@ -213,12 +231,21 @@ private static bool BuildProjectBlocking(BuildInfo buildInfo)
 
             pr.Step("Building project solution", 0);
 
+            string rcBefore = GetRuntimeConfig();
+
             if (!Build(buildInfo))
             {
                 ShowBuildErrorDialog("Failed to build project solution");
                 return false;
             }
 
+            string rcAfter = GetRuntimeConfig();
+
+            if (rcBefore != rcAfter)
```

Comment:
Instead of reading the entire file and comparing them, would it be better to compare modified time or maybe MD5 hashes of the file?